manage_breast_screening/config/settings.py

```diff
@@ -243,4 +248,7 @@ def boolean_env(key, default=None):
 
 AUDIT_EXCLUDED_FIELDS = ["password", "token", "created_at", "updated_at", "id"]
 
-LOGIN_URL = "auth:sign_in"
+if PERSONAS_ENABLED:
+    LOGIN_URL = "demo:persona_login"
+else:
+    LOGIN_URL = "auth:sign_in"
```

Comment:
It's a little unintuitive to have different kinds of authentication split between the auth and demo apps.

Suggestion: we could have all auth-related code in the auth app while still maintaining separation between the logic / URLs for persona vs. CIS2 authentication. For example, `persona_views.py` alongside the existing `views.py`.